### PR TITLE
punct/dating/edits mich 820-945, further hammond revisions

### PR DESCRIPTION
**Pull request by spapando**

### CODE REVIEW

File: xml_inscriptions/transcribed/MA.Glouc.HCM.L.Tmp97.6.8-mdn.xml [hunk 1]
```diff
@@ -144,8 +144,8 @@
                <lb n="3" break="no"/>grippa<supplied reason="omitted">e</supplied></name></persName>, Divi <expan><abbr>A<sic>gu</sic></abbr><ex>usti</ex></expan>
                <lb n="4"/>nepotis, uxor
                <lb n="5" break="no"/>is <persName><name type="cognomen" key="Germanicus">Germanici</name>
-               <lb n="6"/><name type="cognomen" key="Caesar">C<sic>ea</sic>sari</name></persName>, matris
-               <lb n="7"/><persName><name type="praenomen" key="Gaius"><expan><abbr>G</abbr><ex>ai</ex></expan></name> <name type="cognomen" key="Caesar"><expan><abbr>C<sic>ea</sic></abbr><ex>sa</ex><abbr>r</abbr><ex>is</ex></expan></name> <name type="cognomen" key="Augustus"><expan><abbr>Aug</abbr><ex>usti</ex></expan></name>
+               <lb n="6"/><name type="cognomen" key="Caesar">C<sic>ea</sic>sari<supplied reason="omitted">s</supplied></name></persName>, matris
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/MA.Glouc.HCM.L.Tmp97.6.8-mdn.xml:147]: Shouldn't the markup here include that you're supplying from an alternate copy? Should have `<supplied reason="omitted" evidence="parallel">`

Comment (spapando) [reply xml_inscriptions/transcribed/MA.Glouc.HCM.L.Tmp97.6.8-mdn.xml:147]: Yes, I'll go back and fix it.

File: xml_inscriptions/transcribed/MI.AA.UM.KM.L.878.r.xml [hunk 2]
```diff
@@ -132,7 +132,7 @@
         
         
                 <div type="commentary">
-                    <p/>
+                   <p>The inscription on the other is side is MI.AA.UM.KM.L.878.v.</p>
```
Comment (sdigiulio) [inline xml_inscriptions/transcribed/MI.AA.UM.KM.L.878.r.xml:135]: Do we need to note this? I thought this was assumed based on the file-name.

Comment (spapando) [reply xml_inscriptions/transcribed/MI.AA.UM.KM.L.878.r.xml:135]: I think I was told at some point to include it in the commentary, but it does seem redundant.

**Comment by sdigiulio** [comment]
No problem; I already took care of it, just wanted to put it on your radar.

**Comment by jbodel** [comment]
Keep verbal comment as well

**Pull request merged**